client/my-sites/checkout/src/payment-methods/credit-card/credit-card-number-field.tsx

```diff
@@ -71,6 +71,7 @@ export default function CreditCardNumberField( {
 					options={ {
 						style: stripeElementStyle,
 						disabled: isDisabled,
+						showIcon: true,
```

Comment:
Looks good to me! 😄

Reply:
This actually needs to hold off until we fix the styling for the credit card field to remove our custom credit card logos since this prop triggers the component to use the ones provided by Stripe.